sklearn/linear_model/base.py

```diff
@@ -50,6 +50,29 @@ def make_dataset(X, y, sample_weight, random_state=None):
 
     This also returns the ``intercept_decay`` which is different
     for sparse datasets.
+
+    Parameters
+    ----------
+    X : array_like, shape = (n_samples, n_features)
+        Training data
+
+    y : array_like, shape = (n_samples, )
+        Target values.
+
+    sample_weight : numpy array of shape [n_samples, ]
```

Comment:
use () not []

Reply:
I don't think we're consistent about that :-/

Reply:
Yes, that's why I opened #11450